src/main/java/com/nashtech/rookie/yasa/controller/RatingController.java

```diff
@@ -27,6 +27,7 @@ public RatingDto createRating(@RequestBody @Valid CreateRatingDto dto) {
     }
 
     @GetMapping
+    @CrossOrigin
```

Comment:
You can use WebMVCConfigurer to configure global CORS instead of configuring each api in the controller